src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -511,28 +511,28 @@ def update_rem_if_mgmt(self, if_oid, if_name):
             if len(mgmt_ip_str) == 0:
                 # the peer advertise an emtpy mgmt address
                 return
-            time_mark = int(lldp_kvs['lldp_rem_time_mark'])
-            remote_index = int(lldp_kvs['lldp_rem_index'])
-            subtype = self.get_subtype(mgmt_ip_str)
-            ip_hex = self.get_ip_hex(mgmt_ip_str, subtype)
-            if subtype == ManAddrConst.man_addr_subtype_ipv4:
-                addr_subtype_sub_oid = 4
-                mgmt_ip_sub_oid = (addr_subtype_sub_oid, *[int(i) for i in mgmt_ip_str.split('.')])
-            elif subtype == ManAddrConst.man_addr_subtype_ipv6:
-                addr_subtype_sub_oid = 6
-                mgmt_ip_sub_oid = (addr_subtype_sub_oid, *[int(i, 16) if i else 0 for i in mgmt_ip_str.split(':')])
-            else:
-                logger.warning("Invalid management IP {}".format(mgmt_ip_str))
-                return
-            self.if_range.append((time_mark,
-                                  if_oid,
-                                  remote_index,
-                                  subtype,
-                                  *mgmt_ip_sub_oid))
-
-            self.mgmt_ips.update({if_name: {"ip_str": mgmt_ip_str,
-                                            "addr_subtype": subtype,
-                                            "addr_hex": ip_hex}})
+            mgmt_ip_set=set()
+            for mgmt_ip in mgmt_ip_str.split(','):
+                time_mark = int(lldp_kvs['lldp_rem_time_mark'])
+                remote_index = int(lldp_kvs['lldp_rem_index'])
+                subtype = self.get_subtype(mgmt_ip)
+                if not subtype:
+                    logger.warning("Invalid management IP {}".format(mgmt_ip))
+                    continue
+                mgmt_ip_tuple = ip2byte_tuple(mgmt_ip)
+                if mgmt_ip_tuple in mgmt_ip_set:
+                    continue
```

Comment:
if mgmt_ip_set is present in mgmt_ip_tuple, should the data be still updated instead of continuing?
So that the time_mark gets updated?

Reply:
we keep data (eg: time_mark) only for unique ip's. `lldp_rem_time_mark` is associated with all unique ip's in `lldp_rem_man_addr`